crates/runtime/src/dataconnector/databricks.rs

```diff
@@ -172,6 +177,15 @@ async fn get_delta_table(
     };
     storage_options.insert(AWS_S3_ALLOW_UNSAFE_RENAME.to_string(), "true".to_string());
 
+    let timeout: Option<String> = params
+        .as_ref() // &Option<HashMap<String, String>>
+        .as_ref() // Option<&HashMap<String, String>>
+        .and_then(|params| params.get("timeout").cloned());
+
+    if let Some(timeout) = timeout {
+        storage_options.insert("timeout".to_string(), timeout);
```

Comment:
What happens if the timeout value is invalid?

Reply:
Let's increase the default to 60s

Reply:
Do we need this option for S3 as well?

Reply:
> Let's increase the default to 60s
60s is not the magic number. It's rather good to stick the downstream dependency is, and leave the user to override it. Which follows the same pattern delta-rs using object storage.

> Do we need this option for S3 as well?
No need for s3 so far from the testing.

Reply:
> failed to parse "60xxljksd" as Duration

Error propagated from delta-rs.

Reply:
set the default to 300s